config/initializers/apipie.rb

```diff
@@ -19,7 +19,7 @@
     :operatingsystem_families => Operatingsystem.families.join(", "),
     :providers => ComputeResource.providers.join(', '),
     :default_nic_type => InterfaceTypeMapper::DEFAULT_TYPE.humanized_name.downcase,
-    :template_kinds => -> { TemplateKind.scoped.map(&:name).join(", ") },
+    :template_kinds => -> { TemplateKind.where(nil).map(&:name).join(", ") },
```

Comment:
Not that it makes a huge difference as the amount of template kinds is not large, but I would got with `.pluck(:name)` instead of `.map(&:name)`.